### ISSUE TITLE
Pretrain MaskRCNN model

### ISSUE THREAD

**Issue by mbrookhart** [NervanaSystems/ngraph-mxnet#297]
Need to find a model

Acceptance Criteria:
1) Train model to the default number of epochs and within 10% expected accuracy (on GPU)
2) Upload saved model symbol json and parameters to deepmark
3) Provide a script for testing model accuracy on test data.

**Comment by mbrookhart** [comment]
Ruby found https://github.com/TuSimple/mx-maskrcnn

**Comment by ruby-nervana** [comment]
The tusimple implementation above has a custom ROIAlign op and is compiled with an older revision of mxnet `bbf8b3e`. Note that there is a minor error in the `README`: the `v1` ROIAlign files in `rcnn/CXX_OP` should be copied to `src/operator/contrib` while the other ones go to `src/operator`. Was able to compile `make -j 32 USE_OPENCV=1 USE_BLAS=openblas USE_CUDA=1 USE_CUDA_PATH=/usr/local/cuda-9.0 USE_CUDNN=1`, download the pretrained weights, and see qualitatively inference `scripts/demo.sh` works. 

However, the most recent mxnet has a contributed ROIAlign op: https://mxnet.incubator.apache.org/api/python/symbol/contrib.html#mxnet.symbol.contrib.ROIAlign .  Need to understand if we need to use an implementation that uses this ROIAlign or is using the custom implementation in this tusimple repo okay?

**Comment by mbrookhart** [comment]
https://issues.apache.org/jira/browse/MXNET-411

Looks like the one in mxnet is adapted from the tusimple example, probably safe to use either.

**Comment by ruby-nervana** [comment]
Model, data and README are saved in `/nfs/fm/disks/aipg_trained_models_01/mxnet/mask-rcnn`
